woocommerce-gutenberg-products-block.php

```diff
@@ -239,25 +239,38 @@ function woocommerce_blocks_get_i18n_data_json( $translations, $file, $handle, $
 add_filter( 'pre_load_script_translations', 'woocommerce_blocks_get_i18n_data_json', 10, 4 );
 
 /**
- * Filter translations so we can retrieve translations from Core when the original and the translated
- * texts are the same (which happens when translations are missing).
+ * Add notice to the admin dashboard if the plugin is outdated.
  *
- * @param string $translation Translated text based on WC Blocks translations.
- * @param string $text        Text to translate.
- * @param string $domain      The text domain.
- * @return string WC Blocks translation. In case it's the same as $text, Core translation.
+ * @see https://github.com/woocommerce/woocommerce-blocks/issues/5587
  */
-function woocommerce_blocks_get_php_translation_from_core( $translation, $text, $domain ) {
```

Comment:
Is removing this function a mistake, or I'm missing something?

Reply:
PR probably needs to be rebased given this is change in another PR.

Reply:
My bad! Have no ideas why I remove this function in the latest commit.